index.html

```diff
@@ -16,52 +16,54 @@
       <div class="navbar-title">Developer Tools</div>
     </div>
     <div class="techs-container">
-      <h1 class="title">Front-End Tools</h1>
       <div class="techs">
-        <div class="tech">
+        <h1 class="title">Front-End Tools</h1>
+        <div class="tech featured-tech">
           <img src="assets/react_icon.svg" class="tech-logo" alt="React Tech Logo" />
           <h2 class="tech-title">React</h2>
           <span class="tech-release">Release: 2013</span>
           <p class="tech-desc">
             React is a declarative, component-based JavaScript library for building user interfaces.
           </p>
         </div>
-        <div class="tech">
-          <img src="assets/angular_icon.svg" class="tech-logo" alt="AngularJS Tech Logo" />
-          <h2 class="tech-title">AngularJS</h2>
-          <span class="tech-release">Release: 2010</span>
-          <p class="tech-desc">
-            AngularJS (commonly referred to as AngularJS 1.X) is a JavaScript framework for building SPAs with an MVC Architecture
-          </p>
-        </div>
-        <div class="tech">
-          <img src="assets/vue_icon.svg" class="tech-logo" alt="VueJS Tech Logo" />
-          <h2 class="tech-title">VueJS</h2>
-          <span class="tech-release">Release: 2014</span>
-          <p class="tech-desc">
-            VueJS (pronounced /vju:/, like view) is an open-source progressive JavaScript framework for building user interfaces.
-          </p>
-        </div>
-        <div class="tech">
-          <img src="assets/elm_icon.svg" class="tech-logo" alt="Elm Tech Logo" />
-          <h2 class="tech-title">Elm</h2>
-          <span class="tech-release">Release: 2016</span>
-          <p class="tech-desc">
-            Elm is a domain-specific functional programming language for declaratively creating web browser-based graphical user interfaces.
-          </p>
-        </div>
-        <div class="tech">
-          <img src="assets/polymer_icon.svg" class="tech-logo" alt="Polymer Tech Logo" />
-          <h2 class="tech-title">Polymer</h2>
-          <span class="tech-release">Release: 2015</span>
-          <p class="tech-desc">
-            Polymer is an open-source JavaScript library for building web applications using Web Components.
-          </p>
+        <div class="other-techs">
```

Comment:
this is unnecessary

Reply:
you can set this "other-tech" to the common tech and override it for the first one in featured-tech